src/desktop/apps/editorial_features/components/venice_2017/client/index.coffee

```diff
@@ -135,28 +133,6 @@ module.exports = class VeniceView extends Backbone.View
     else
       sd.APP_URL + section.video_url
 
-  showCta: (e) ->
```

Comment:
This code was actually not accessible anymore, it was a state used for unpublished videos -- now all videos are published and the CTA will never display.